src/components/TruncatedText/index.test.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { GetModelPlanBaseDocument } from 'gql/generated/graphql';
+import VerboseMockedProvider from 'tests/MockedProvider';
+
+import TruncatedText from './index';
+
+const mocks = [
+  {
+    request: {
+      query: GetModelPlanBaseDocument,
+      variables: { id: 'f11eb129-2c80-4080-9440-439cbe1a286f' }
+    },
+    result: {
+      data: {
+        modelPlan: {}
+      }
+    }
+  }
+];
```

Comment:
Test throw error if Provider (and therefore mocks) were not included in test. Is there another way to solve this? Currently, this is just a random mock and not used in `TruncatedText` component

Reply:
That's the incorrect query to mock.  TruncatedText utilized the `MentionTextArea`.  We might actually might want to not use this.  The `MetionTextArea` has a query embedded within it - `useSearchOktaUsersLazyQuery`.  I forgot why I decided to utilize `MentionTextArea`.  I don't think that was the right choice, considering how much overhead that component has built in.  We should probably try to build `TruncatedText` without it.  Sorry this was my fault